zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/internal/fetcher/HttpFetcher.kt

```diff
@@ -50,19 +50,22 @@ internal class HttpFetcher(
     url: String,
     freshAtEpochMs: Long,
   ): LoadedManifest {
-    val manifestBytesWithoutBaseUrl = fetchByteString(applicationName, null, url)
+    val manifestBytesWithoutBaseUrlUtf8 = fetchByteString(applicationName, null, url)
```

Comment:
How about we just limit the download size here somehow ? We should not download it in the first place if it's > MANIFEST_MAX_SIZE.